jni_gen/examples/pdfbox_example/pdfbox_plugin/pubspec.yaml

```diff
@@ -0,0 +1,39 @@
+name: pdfbox_plugin
+description: |
+  Example of using jni_gen to generate bindings for a non-trivial library
+version: 0.0.1
+publish_to: none
+homepage: https://github.com/dart-lang/jni_gen
+
+environment:
+  sdk: ">=2.17.6 <3.0.0"
+  #flutter: ">=2.11.0"
+
+dependencies:
+  jni:
+    path: ../../../../jni/
+  #flutter:
+    #sdk: flutter
+  plugin_platform_interface: ^2.0.2
+
+dev_dependencies:
+  ## Path dependency for sake of the example
+  jni_gen:
+    path: ../../../
+  flutter_test:
```

Comment:
```
$ dart pub get
Resolving dependencies... 
Because pdfbox_plugin_example depends on flutter_test from sdk which doesn't exist (the Flutter SDK is not available), version solving failed.

Flutter users should run `flutter pub get` instead of `dart pub get`.
```